src/skmultiflow/drift_detection/base_drift_detector.py

```diff
@@ -1,8 +1,8 @@
 from abc import ABCMeta, abstractmethod
-from skmultiflow.core.base_object import BaseObject
+from skmultiflow.core import BaseStreamEstimator
 
 
-class BaseDriftDetector(BaseObject, metaclass=ABCMeta):
+class BaseDriftDetector(BaseStreamEstimator, metaclass=ABCMeta):
```

Comment:
Is a `DriftDetector` really a `StreamEstimator` ? 
If I try to implement its `partial_fit` and `predict`methods, it gives something like:
```python
def partial_fit(self, X):
    for value in X:
        self.add_element(X)
    return self

def predict(self):
    return self.detected_change()   
```

with no input for predict I don't think it is an "estimator"

Reply:
`fit`, `partial_fit, `predict` and `predict_proba` are now moved into task-specific mixins.

In this case, making the `BaseDriftDetector` class an extension of `BaseStreamEstimator` does not imply this extra functionality.

However, as previously mentioned, the name is confusing. We might rename `BaseStreamEstimator` to `BaseObject` for clarity.